openfisca_core/tests/assets/parameters.yaml

```diff
@@ -0,0 +1,124 @@
+
+# Regular parameter values : meta + VALUES handling variation of its value in time
+- variable: smic_horaire_brut
```

Comment:
There are some variable's formula using more than one parameters and there are parameters entering many formulas and extension etc. We should leave that to the parsers in my humble opinion.